### ISSUE TITLE
item.url does not contain /index.html anymore

### ISSUE THREAD

**Issue by tkrotoff** [octopress/minify-html#8]
minify-html was working fine for me before, it fails silently now I guess due to some recent changes (weeks or months ago?) in Jekyll.

Explanations:

File [octopress-minify-html.rb](https://github.com/octopress/minify-html/blob/v1.2.2/lib/octopress-minify-html.rb#L13)
```Ruby
def minify?(item)
  if item.url.end_with?('html')
    # ...
  end
end
```

`item.url` does not end with .html thanks to [Jekyll pretty permalinks](http://jekyllrb.com/docs/permalinks/#built-in-permalink-styles).

A possible fix:
```Ruby
def minify?(item)
  path = item.url
  path = File.join(path, "index.html") if File.extname(path).empty?
  if path.end_with?('html')
    # ...
  end
end
```

I guess `item.url` contained `/index.html` before.

Currently the only file minify-html treats is my `404.html` :)

**Comment by tkrotoff** [comment]
btw, in order to detect this kind of failure and give feedback to the user, I suggest to add a verbose mode:

```Ruby
def post_render(item)
  if minify?(item)
    item.output = HtmlPress.press(item.output)
    puts "#{item.url} minified"
   end
end
```

Output:
```
$ bundle exec rake build[production]
Configuration file: /Users/tanguy/osteo15.com/_config.yml
/2013/11/22/Interview-sur-l'osteopathie-et-les-femmes-enceintes/ minified
/2013/12/10/Emission-sur-la-grippe/ minified
/2014/01/27/Prise-en-charge-globale-du-patient/ minified
[...]
/404.html minified
/Contact/ minified
/Liens/ minified
/blog/ minified
/ minified
/tags/ minified
```

**Comment by parkr** [comment]
it should use `item.destination(dest_dir)` instead of `item.url`

**Comment by imathis** [comment]
I just released version 1.2.3 with this fix. Thanks @parkr!

**Comment by tkrotoff** [comment]
:+1: tested, works fine, thx
